### PR TITLE
Debt/oud besturen

### PR DESCRIPTION
**Pull request by Phavion**
Not sure yet about the styling. Would love opinions on how to style the board pages.

### CODE REVIEW

File: src/components/Board.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components'
 
 const Board = ({ board }) => (
   <BoardCard color={board.color}>
-    <h3>{board.number}e bestuur</h3>
+    <h3><a href={'/besturen/' + board.slug.replace(/\W+/g, '-').toLowerCase()}>{board.number}e bestuur</a></h3>
```
Comment (emielvanrijn) [inline src/components/Board.jsx]: The regex replace is unnecessary in combination with slug, as slug is being edited to be this compliant in Contentful by default :)

File: src/pages/vereniging/besturen.jsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ const Bestuur = (props) => {
 
 
 const BoardsList = styled.div`
+  margin-top: 10pt;
```
Comment (emielvanrijn) [inline src/pages/vereniging/besturen.jsx]: What is this 'pt' thingy? Maybe we can better use 'em' for units, as this is the most used relative unit accroding to [MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Values_and_units)

File: src/components/Board.jsx (hunk 1 above)
Comment (Phavion) [reply src/components/Board.jsx]: will remove it :)

File: src/pages/vereniging/besturen.jsx (hunk 2 above)
Comment (Phavion) [reply src/pages/vereniging/besturen.jsx]: A thing @Agarwaemben pointed me to :)

**Pull request merged**